src/locales/en-US/commands.json

```diff
@@ -1473,5 +1473,19 @@
     "commandDescription": "Searches for some image on google",
     "commandUsage": "<query>",
     "noQuery": "Please insert something to search."
+  },
+  "move": {
+    "commandDescription": "Moves a message from a channel to another.",
+    "commandUsage": "<messageLink> <channelMention>",
+    "missingMessageLink": "You forgot to send the message link, right click a message and click Copy Message Link, you might need developer mode on.",
+    "missingChannelMention": "Please mention a channel.",
+    "invalidMessageLink": "Invalid Message Link",
```

Comment:
This is not being used anywhere

Reply:
Line 12 of move.js
`missingError: 'commands:move.missingMessageLink'`

Line 14 of move.js
`missingError: 'commands:move.missingMessageLink`

You're right about invalidMessageLink.

Reply:
This comment refers to `invalidMessageLink` only. Accept this suggestion to remove it:

```suggestion
```

Reply:
Lol, thanks github for showing all 4 lines then. I already changed many lines on my local file so I'll do all the changes here and commit them all in a instant.